internal/report/report.go

```diff
@@ -386,7 +386,7 @@ func PrintAssembly(w io.Writer, rpt *Report, obj plugin.ObjTool, maxFuncs int) e
 	symNodes := nodesPerSymbol(g.Nodes, symbols)
 
 	// Sort for printing.
-	var syms []*objSymbol
+	syms := make([]*objSymbol, 0, len(symNodes))
```

Comment:
I believe using creates an array of the specified size filled with empty values, then appending will add a non-empty value to the end (rather filling in an empty spot).
For example
```
arr := make([]int, 2)
arr = append(arr, 1)
```
produces the array [0, 0, 1].

Reply:
@nolanmar511, regarding the example that you have provided, yes that would produce [0, 0, 1].
But, the make() that I have used has length=0 and capacity=len(symNodes), so it's different from giving it a length of 2.

So, the equivalent example = 
```
arr := make([]int, 0, 5)
arr = append(arr, 1)
```

which produces the array [1]. Playground link: https://play.golang.org/p/nSUHnfqubiJ